### PR TITLE
Added reload insance method

### PR DESCRIPTION
**Pull request by outluch**

### CODE REVIEW

File: 08-Lucid-ORM/01-Getting-Started.adoc [hunk 1]
```diff
@@ -432,6 +432,25 @@ const User = use('App/Models/User')
 const users = await User.truncate()
 ----
 
+
+== Instance methods
+Lucid models come with a bunch of instance methods to do common operations without using the query builder interface.
+
+==== reload
+Reload model from database.
+Example use case: you have a model with some hook on creation (eg. set some field to some uuid). To get that value in your model instance object you need to reload model from database.
```
Comment (RomainLanz) [inline 08-Lucid-ORM/01-Getting-Started.adoc]: You got some typo here. 😄 

```suggestion
Example use case: you have a model with some hooks on creation. To get that value in your model instance you need to reload the model from the database.
```

File: 08-Lucid-ORM/01-Getting-Started.adoc [hunk 2]
```diff
@@ -432,6 +432,25 @@ const User = use('App/Models/User')
 const users = await User.truncate()
 ----
 
+
+== Instance methods
```
[the remark this answers is not in the document]
Comment (RomainLanz) [reply 08-Lucid-ORM/01-Getting-Started.adoc]: It's already the case

[the remark this answers is not in the document]
Comment (outluch) [reply 08-Lucid-ORM/01-Getting-Started.adoc]: But you mentioned 4.0 docs, and I suggested changes in 4.1 last version. There are a lot of reorganization. Check plz

**Comment by RomainLanz** [comment]
Thanks 😄

**Pull request merged**